.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,23 +10,30 @@ assignees: ''
 ### Bug Description
 A clear and concise description of what the bug is.
 
+### General Information
+Affected endpoint(s) [e.g. /contributions/count].
+Affected parameter(s) [e.g. bpolys, format].
+HTTP method used [e.g. POST].
+Used tool/library for the request [e.g. cURL, Postman].
```

Comment:
Please have a look at the [rendered version](https://github.com/GIScience/ohsome-api/blob/issues-template/.github/ISSUE_TEMPLATE/bug_report.md#general-information) of this markdown file as it gives an indicator how it looks like in the later issue. I would suggest to use bulletpoints as within _System Information_.